live-examples/js-examples/globalprops-undefined.html

```diff
@@ -0,0 +1,14 @@
+<pre>
+<code id="static-js">function test(t) {
+  if (t === undefined) {
+     return "Undefined value!";
+  }
+  return t;
+}
+
+var x;
+
+console.log(test(x));
+// expected output: 0
```

Comment:
Should be:

`// expected output: ""Undefined value!"`

Reply:
:+1: